client/components/addons/addon-collection/addon-collection.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const AddonCollection = ({ addons }) => {
+	const renderedAddonElements = addons.map(addon => (
+		<div id={addon.name} key={addon.name}>
```

Comment:
id is for nothing right now - I'll remove it as it has no use. Will also update to unordered list